.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Why do we have two files for feature requests?

Reply:
my bad! I believe it was autogenereated, ill delete! good catch! 👍